UI/EmuScreen.cpp

```diff
@@ -405,6 +405,7 @@ void EmuScreen::bootComplete() {
 EmuScreen::~EmuScreen() {
 	if (!invalid_ || bootPending_) {
 		// If we were invalid, it would already be shutdown.
+		Achievements::UnloadGame();
```

Comment:
Does this rather belong just in `PSP_Shutdown()` itself?  It sorta seems like that might be better (also PSP_Init() somewhere for SetGame.)  That said, not sure how this interacts with libretro.

-[Unknown]

Reply:
Yeah, it probably done.